docs/src/man/basics.md

```diff
@@ -0,0 +1,2002 @@
+    6 │     5  male        1  free     NA               NA
+    7 │     6  male        2  own      quite rich       NA
+    8 │     7  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮       ⋮       ⋮            ⋮                ⋮               ⋱
+  994 │   993  male        3  own      little           little                 ⋯
+  995 │   994  male        2  own      NA               NA
+  996 │   995  female      1  own      little           NA
+  997 │   996  male        3  own      little           little
+  998 │   997  male        2  own      little           NA                     ⋯
+  999 │   998  male        2  free     little           little
+ 1000 │   999  male        2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+
+julia> select(german, r"Sex") # select columns containing 'Sex' string
+1000×1 DataFrame
+  Row │ Sex
+      │ String
+──────┼────────
+    1 │ male
+    2 │ female
+    3 │ male
+    4 │ male
+    5 │ male
+    6 │ male
+    7 │ male
+    8 │ male
+  ⋮   │   ⋮
+  994 │ male
+  995 │ male
+  996 │ female
+  997 │ male
+  998 │ male
+  999 │ male
+ 1000 │ male
+985 rows omitted
+
+julia> select(german, r"Sex", "Job") # Selects more than one column
+1000×2 DataFrame
+  Row │ Sex     Job
+      │ String  Int64
+──────┼───────────────
+    1 │ male        2
+    2 │ female      2
+    3 │ male        1
+    4 │ male        2
+    5 │ male        2
+    6 │ male        1
+    7 │ male        2
+    8 │ male        3
+  ⋮   │   ⋮       ⋮
+  994 │ male        3
+  995 │ male        2
+  996 │ female      1
+  997 │ male        3
+  998 │ male        2
+  999 │ male        2
+ 1000 │ male        2
+      985 rows omitted
+
+julia> select(german, :Sex => :x1, :Age => :x2) # rename columns
+1000×2 DataFrame
+  Row │ x1      x2
+      │ String  Int64
+──────┼───────────────
+    1 │ male       67
+    2 │ female     22
+    3 │ male       49
+    4 │ male       45
+    5 │ male       53
+    6 │ male       35
+    7 │ male       53
+    8 │ male       35
+  ⋮   │   ⋮       ⋮
+  994 │ male       30
+  995 │ male       50
+  996 │ female     31
+  997 │ male       40
+  998 │ male       38
+  999 │ male       23
+ 1000 │ male       27
+      985 rows omitted
+
+julia> select(german, :Age, :Age => ByRow(sqrt)) # transform columns by row
+1000×2 DataFrame
+  Row │ Age    Age_sqrt
+      │ Int64  Float64
+──────┼─────────────────
+    1 │    67   8.18535
+    2 │    22   4.69042
+    3 │    49   7.0
+    4 │    45   6.7082
+    5 │    53   7.28011
+    6 │    35   5.91608
+    7 │    53   7.28011
+    8 │    35   5.91608
+  ⋮   │   ⋮       ⋮
+  994 │    30   5.47723
+  995 │    50   7.07107
+  996 │    31   5.56776
+  997 │    40   6.32456
+  998 │    38   6.16441
+  999 │    23   4.79583
+ 1000 │    27   5.19615
+        985 rows omitted
+```
+
+In the above example `ByRow` type is a special type used for selection operations to signal that
```

Comment:
you have already used it several examples above. Can you please reorder the content?